documentation/get-started/tutorials/get-started-qfc.en.md

```diff
@@ -230,6 +230,26 @@ Once configured, you can press the cloud button to open the synchronization dial
 
 Now you should see your project and files on [QFieldCloud](https://app.qfield.cloud/)
 
+### Activate Notifications for QFieldCloud changes
+
+1. Access the Settings of your account.
+2. Navigate to the Notifications section. Here, you can customize the frequency of notifications you wish to receive at the valid email address registered with your account.![Synchronize](../../assets/images/frequency_notifications_settings.png)
```

Comment:
```suggestion
2. Navigate to the Notifications section. Here, you can customize the frequency of notifications you wish to receive at the email address registered with your account.![Synchronize](../../assets/images/frequency_notifications_settings.png)
```

Reply:
Please also include this: 
... does only notify you of events where you are not the actor. You only get notifications for events triggered by your organization members or project collaborators.

@SeqLaz maybe try out collaborators, I know org members work. I did not test if this really works as intended ...


related code snipped:
```
def _concerned_users_in_entity(entity: User):
    """Returns a list of users (of User.Type.PERSON) concerned by updates to an user (any type)"""

    return Person.objects.for_entity(entity)


def _concerned_users_in_project(project: Project):
    """Returns a list of users concerned by updates to a project"""

    return Person.objects.for_project(project).exclude(
        project_role_origin=ProjectQueryset.RoleOrigins.PUBLIC
    )
```

Find it in the repo:
https://github.com/opengisch/qfieldcloud/blob/bbe84ed68372f95ffa73b7f3e186b7d378f439bf/docker-app/qfieldcloud/notifs/signals.py#L24

Reply:
Done, the functionality in staging works